docs-content/release-notes.html.md.erb

```diff
@@ -5,6 +5,31 @@ owner: Partners
 
 These are release notes for CyberArk Conjur Service Broker for VMware Tanzu
 
+## <a id="ver"></a> v1.2.0
+
+**Release Date:** July 27, 2020
+
+Features in this release:
+
+* Improved service-broker validation and error handling
+  <br/>
+  - Service broker returns 404 when the org or space policy branches do not exist
+    as expected with a helpful error message, rather than returning 500.
+  - Service broker health check verifies the `CONJUR_POLICY` exists on the server, if set.
+
+* Expanded buildpack configuration capabilities 
+  <br/>
+  - The runtime location for `secrets.yml` can now be configured by setting the
+    `SECRETS_YAML_PATH` environment variable for the Cloud Foundry application.
+  - Buildpack supply step now scans build directory for candidate secrets.yml files
+    and reports them to the output.
+
+* The buildpack now properly reads Conjur credentials from `VCAP_SERVICES` when
+  `VCAP_SERVICES` contains credentials for other services with the same field
```

Comment:
`service` is a loaded term in Cloud Foundry, so bear with me.

`VCAP_SERVICES` is an environment variable that lists configuration information for any "service" that your application is "bound to". When your application "binds" to a Conjur "service instance", the Conjur service broker returns Conjur credentials that are injected into the application's `VCAP_SERVICES` variable.

If your app is bound to multiple services, you will have credentials / config for `cyberark-conjur` in your VCAP_SERVICES, but you may also have credentials for other services and some could use the same keys. Previously we just pulled a value for the `version` key; with this update, we pull _just_ the subset of the json under the `cyberark-conjur` and retrieve the `version` key from the `cyberark-conjur` part of the json tree.

Hopefully that makes some sense; if there's any more confusion, the PR that fixed this is here and may shed more light: https://github.com/cyberark/cloudfoundry-conjur-buildpack/pull/49

Given this info, do you have a suggested alternative description

Reply:
Yup, makes more sense now.  Maybe:

> The buildpack now properly reads only the Conjur credentials from `VCAP_SERVICES`.   Previously, it could consume credentials for other services, if their field names exactly matched those used by Conjur (e.g. `version` is a very common field).

Reply:
@BradleyBoutcher this suggestion still needs to be implemented ^